_posts/2020-01-29-Introduction_to_the_IAC_Team.md

```diff
@@ -0,0 +1,121 @@
+Usually in Office Hours, we are used to fielding questions from folks who are trying to enhance functionality in a module, determine the best way to test or maybe they want to know how to get started to write a new module from scratch.
+This one was a little bit different: 'Who are the IAC Team?' This was followed up with some more questions: 'What is your development process? What are the tools you use day-to-day?', and my favourite, 'Who comes up with those awesome meeting room names?'.
+I hope I answered that particular community member's questions, but it prompted a discussion amongst the team about whether the people within the Puppet Community know who we are, what we do, how we can help you and how to reach us if you need that help.
+ 
+## Who are we?
+Firstly, an introduction - the Infrastructure Automation Content (IAC) team spans across 3 time zones, from East to West we have three members in Timișoara, Romania, six members in Belfast, Northern Ireland (including our Engineering Manager) and finally, one man and his dog holding the fort in St. Louis, Missouri, USA.
+We also work closely with Product Management teams both in our Portland, Oregon HQ and Belfast.
+ 
+## What do we do?
+The IAC team are responsible for developing and maintaining tooling that contributors to Puppet can use to enhance existing or add new modules to Puppet.
+We are also responsible for reviewing the submissions from the community, assisting in development and testing efforts, if required and releasing new versions of modules to the Forge.
+ 
+ 
+## Workflow and Processes
+### Work Prioritisation, Ticketing and Development
+We follow the KanBan model of work prioritisation and categorisation.
+We have chosen JIRA as our ticketing system. Within JIRA, the IAC Content team has a number of Epics which we categorise work under.
+A few examples of day-to-day Epics we interact with a lot:
+- Customer Escalations
+- Triage Tasks (more about Triage later!)
+ 
+We also have Epics for larger pieces of work, for example:
+- PowerShell in Puppet
+- Automated Gem & Testing Release
+ 
+We also follow the AGILE methodology of having a stand-up meeting daily, where all members describe what they have been working on since the last meeting and what they will work on after until the next meeting.
+Its also during this meeting that we can attempt to unblock individuals who are stuck - either during the stand-up or by assigning time at some stage after. We're in constant communication throughout the day, either face-to-face or over Slack.
+We can also jump in to meeting rooms / booths and chat to our colleagues in other time zones and screen share.
+So it's never the case that we're stuck for very long - we all like to have each other's backs and help at short notice.
+
+Each member is responsible for the end-to-end delivery of a specific work item.
+So the planning, development, testing, documentation, etc... will all be done by one individual.
```

Comment:
I don't _think_ this is strictly true - I've written up tickets and epics for other folx to follow through on and gotten them that way too - but this definitely _does_ happen!

🤔

Reply:
Yeah, fair enough - I'll remove that